lib/jobs/ansible-job.js

```diff
@@ -78,10 +95,23 @@ function ansibleJobFactory(
      * @memberOf AnsibleJob
      */
      AnsibleJob.prototype._cleanup = function cleanup() {
+        var self = this;
+        try {
+            var data = fs.readFileSync(self.outFileName, 'utf8');
+            self.context.ansibleResultFile.push(data);
+            fs.unlink(self.outFileName, function(err) {
+                if (err) {
+                    logger.warning("Uable to remove temporary Ansible " +
+                        "result file " + self.outFileName + " " + err);
+                }
+            });
+        } catch (e) {
+            logger.info("Error reading Ansible result file " +
```

Comment:
I don't think it should be an error, since the Ansible playbook may not always write a result file (e.g.: if there is no result to return).  So it is not an error if the file is not present.  Perhaps it should be a warning?